src/components/LoginLoginLogin.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div>
+    <section class="gradient-custom" style="background: #6a11cb; height: 100vh">
```

Comment:
Where did you define the `gradient-custom` class? I do not think it is a Bootstrap in-built class.